src/js/slider.js

```diff
@@ -11,7 +11,9 @@
                 model: '=',
                 min: '@',
                 max: '@',
-                step: '@'
+                step: '@',
+                shouldSnap: '@',
+                snapRange: '@'
```

Comment:
These should be named so that people understand that they are controlling the snap to the initial position and not the snap to step